benchexec/container.py

```diff
@@ -746,6 +805,41 @@ def escape(s):
     )
 
 
+def make_fuse_overlay_mount(exe, mount, lower, upper, work):
+    logging.debug(
+        "Creating overlay mount with fuse-overlayfs: target=%s, lower=%s, upper=%s, work=%s",
+        mount,
+        lower,
+        upper,
+        work,
+    )
+
+    def escape(s):
+        return s.replace(b"\\", rb"\\").replace(b":", rb"\:").replace(b",", rb"\,")
+
+    cmd = (
+        exe,
+        b"-o",
+        b"lowerdir="
+        + escape(lower)
+        + b",upperdir="
+        + escape(upper)
+        + b",workdir="
+        + escape(work),
+        escape(mount),
+    )
+
+    try:
+        result = subprocess.run(
+            args=cmd, check=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT
+        )
+        if result.stdout:
+            logging.debug("fuse-overlayfs: %s", result.stdout.decode())
+    except subprocess.CalledProcessError as e:
+        logging.error("Error executing command: %s\n, %s", e, e.stderr.decode())
+        sys.exit(1)
```

Comment:
Will `e.stderr` have a meaningful value if we use `stderr=subprocess.STDOUT`? I suspect it is empty.

When you catch an error and exit, it is better to keep that together instead of logging the error message separately. I would suggest to pass an appropriate error message to `sys.exit()` instead of the logging.

Reply:
plz review this:

https://github.com/sosy-lab/benchexec/blob/195e4d06c3db6b042ec80239acada55927a27d7f/benchexec/container.py#L840-L850